memiavl/db.go

```diff
@@ -284,6 +284,9 @@ func (db *DB) pruneSnapshots() {
 		}
 
 		counter := db.snapshotKeepRecent
+		if counter == 0 {
```

Comment:
Keep all is not a viable option for memiavl, because disk space will skyrocketing

Reply:
I see, not expected